README.md

```diff
@@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [BetterTouchTool](http://blog.boastr.net/)
   - [BibDesk](http://bibdesk.sourceforge.net/)
   - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
+  - [Bitchx](https://github.com/BitchX/)
```

Comment:
That should have everything fixed now. 

-- 
Troy Wilson
PGP Signed Emails and PGP Encryption available upon request.



On April 17, 2015 at 5:36:52 PM, Laurent Raufaste (notifications@github.com) wrote:

In README.md:

> @@ -154,6 +154,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
>    - [BetterTouchTool](http://blog.boastr.net/)
>    - [BibDesk](http://bibdesk.sourceforge.net/)
>    - [Billings Pro Server Admin](https://www.marketcircle.com/billingspro/download/billingspro-server/)
> +  - [Bitchx](https://github.com/BitchX/)
I don't mind which one, but it should be the official one.
https://github.com/BitchX/ looks less updated than http://www.bitchx.org/
It's a mess =)

—
Reply to this email directly or view it on GitHub.